### DISCUSSION TITLE
COmmand line

### DISCUSSION THREAD

**Discussion by mike37510** [unsound/hfsexplorer#41] [General]
Hi, !!!
An option exists in the interface or on the command line to automatically rename problematic files.
I don't want to stay 8 hours in front of my computer validating each file;)
Thanks

**Comment by josejmm** [comment]
Hi,
In the attached screenshots files, you find the way to do it. It works for me.

![Screenshot_1](https://github.com/user-attachments/assets/2fddddf3-711a-4790-8032-c37d4e87e532)
![Screenshot_2](https://github.com/user-attachments/assets/55b8a38c-0384-4ef2-a3fb-449548e9e0d7)

An awesome improvement will be to save a configuration file, so we don't have to do it each time we ask for an extraction.

Many thanks for your help. 
HFSexplorer is awesome.

**Comment by unsound** [comment]
I opened an issue to track this request: #52

**Comment by unsound** [comment]
Issue #52 has now been closed as the `-auto-rename` option is part of the latest release 2026.4.9.
